cdap-examples/WordCount/src/main/java/co/cask/cdap/examples/wordcount/WordCounter.java

```diff
@@ -41,15 +41,21 @@
  */
 public class WordCounter extends AbstractFlow {
 
+  private final WordCount.WordCountConfig config;
+
+  public WordCounter(WordCount.WordCountConfig config) {
+    this.config = config;
```

Comment:
It is documented. Needs a bit more explanation, maybe? http://docs.cask.co/cdap/current/en/developers-manual/advanced/best-practices.html?highlight=property%20annotation.